modules/backend/controllers/index/index.htm

```diff
@@ -1,21 +1,21 @@
 <?= Form::open(['class'=>'layout-relative dashboard-container']) ?>
     <div id="dashReportContainer" class="report-container loading">
-
         <!-- Loading -->
         <div class="loading-indicator-container">
             <div class="loading-indicator indicator-center">
                 <span></span>
                 <div><?= e(trans('backend::lang.list.loading')) ?></div>
             </div>
         </div>
-
     </div>
 <?= Form::close() ?>
 
 <script>
-    $(document).ready(function() {
-        $.request('onInitReportContainer').done(function() {
-            $('#dashReportContainer').removeClass('loading')
-        })
-    })
+Snowboard.ready(() => {
+    Snowboard.request(null, 'onInitReportContainer', {
+        success: () => {
+            $('#dashReportContainer').removeClass('loading');
+        },
+    });
+});
```

Comment:
Is there a way that we can move this into the scripts block instead so that it doesn't force the script tag to be injected adhoc in the middle of the body element? This causes problems with wrapping the entire contents of the backend in a Vue instance; although that can be worked around it'd be nice to encourage scripts to be put in the same place together at the end of the body tag.

Reply:
I can do it for this instance, but I believe there's quite a lot of inline scripts littered throughout the Backend. I found it easier to focus Vue on a certain section as opposed to making the entire Backend a Vue instance. I think if we wanted to go down that road, we'll have to significantly rewrite the Backend.